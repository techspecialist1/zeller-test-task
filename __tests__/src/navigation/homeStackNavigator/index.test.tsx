jest.useFakeTimers();
import React from 'react';
import renderer, {act} from 'react-test-renderer';
import {NavigationContainer} from '@react-navigation/native';
import HomeStackNavigator from '../../../../src/navigation/homeStackNavigator';

describe('HomeStackNavigator', () => {
  test('renders HomeStackNavigator correctly', () => {
    act(() => {
      const component = renderer.create(
        <NavigationContainer>
          <HomeStackNavigator />
        </NavigationContainer>,
      );

      const tree = component.toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
