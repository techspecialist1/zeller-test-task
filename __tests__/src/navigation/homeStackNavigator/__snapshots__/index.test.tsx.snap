// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeStackNavigator renders HomeStackNavigator correctly 1`] = `null`;
